docs/source/release/v3.8.0/ui.rst

```diff
@@ -22,6 +22,12 @@ User Interface
 Instrument View
 ###############
 
+Workspace Matrix View
+#####################
+* Masked spectra will be highlighted with background colour (typically light grey).
+* Monitor spectra will have a dynamic background colour depending on the system settings (typically light yellow, as before).
+.. figure::  ../../images/MaskedAndMonitor.png
+
```

Comment:
In that case the second line in the release notes should begin with "Unmasked monitor spectra" or something like it.